3-JSS/src/config.js

```diff
@@ -0,0 +1,5 @@
+
+
+export const config = {
+    server: "http://localhost:3000/"
```

Comment:
you don't need to have the prefix in your client code. when u issue a request to the server it always takes the domain + port

Reply:
use instead webpack proxy.